cdc/owner/feed_state_manager.go

```diff
@@ -63,11 +63,15 @@ type feedStateManager struct {
 // newFeedStateManager creates feedStateManager and initialize the exponential backoff
 func newFeedStateManager() *feedStateManager {
 	f := new(feedStateManager)
+	serverConfig := config.GetGlobalServerConfig()
 
 	f.errBackoff = backoff.NewExponentialBackOff()
 	f.errBackoff.InitialInterval = defaultBackoffInitInterval
 	f.errBackoff.MaxInterval = defaultBackoffMaxInterval
-	f.errBackoff.MaxElapsedTime = defaultBackoffMaxElapsedTime
+	if serverConfig.MaxElapsedTime != 90 {
```

Comment:
Why do we use a magic number here instead of defining a constant?

Reply:
By the way, if the user explicitly sets the value to `90m`, we will output the warning log, which will be weird.